.github/workflows/docker_publish.yml

```
@@ -1,68 +1,85 @@
+          export VERSION=${GITHUB_REF#refs/tags/}
+          echo "::set-output name=VERSION_TAG::${VERSION:1}"
+          echo "::set-output name=VERSION::${VERSION}"
 
       - name: Get version
         run: echo "Version is ${{ steps.get-commit-tag.outputs.VERSION }}"
 
       - name: Set production tag
         id: get-prod-tags
         run: |
-            if [ "${{ steps.get-commit-tag.outputs.VERSION }}" == "${{ steps.get-latest-tag.outputs.tag }}" ]
-            then
-              echo "::set-output name=LATEST::memgraph/memgraph-mage:latest"
-            fi
+          if [ "${{ steps.get-commit-tag.outputs.VERSION }}" == "${{ steps.get-latest-tag.outputs.tag }}" ]
+          then
+            echo "::set-output name=LATEST::memgraph/memgraph-mage:latest"
+          fi
 
       - run: echo "Additional tag for production image - ${{ steps.get-prod-tags.outputs.LATEST }}"
 
       - name: Log in to Docker Hub
         uses: docker/login-action@v1
         with:
-            username: ${{ secrets.DOCKER_USERNAME }}
-            password: ${{ secrets.DOCKER_PASSWORD }}
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
 
-      - name: Push production image to Docker Hub
-        id: docker-build-prod
-        uses: docker/build-push-action@v2
-        with:
-          push: true
-          tags: memgraph/memgraph-mage:${{ steps.get-commit-tag.outputs.VERSION_TAG }}, ${{ steps.get-prod-tags.outputs.LATEST }}
-          target: prod
-          file: ./Dockerfile
+      - name: Download memgraph binary
+        run: |
+          curl -L https://download.memgraph.com/memgraph/v${MEMGRAPH_VERSION}/debian-11/memgraph_${MEMGRAPH_VERSION}-1_amd64.deb > memgraph-amd64.deb
+          curl -L https://download.memgraph.com/memgraph/v${MEMGRAPH_VERSION}/debian-11-aarch64/memgraph_${MEMGRAPH_VERSION}-1_aarch64.deb > memgraph-arm64.deb
 
-      - name: Push development image to Docker Hub
-        id: docker-build-dev
-        uses: docker/build-push-action@v2
-        with:
-            push: true
-            tags: memgraph/memgraph-mage:${{ steps.get-commit-tag.outputs.VERSION_TAG }}-dev
-            target: dev
-            file: ./Dockerfile
+      - name: Build & push prod docker images
+        run: |
+          docker buildx build \
+          --target prod \
+          --platform linux/amd64,linux/arm64 \
+          --tag $DOCKER_ORGANIZATION_NAME/$DOCKER_REPOSITORY_NAME:${{ github.event.inputs.version }} \
+          --tag $DOCKER_ORGANIZATION_NAME/$DOCKER_REPOSITORY_NAME:latest \
+          --file Dockerfile.release
+          --push .
+
+      - name: Build & push dev docker images
+        run: |
+          docker buildx build \
+          --target dev \
+          --platform linux/amd64,linux/arm64 \
+          --tag $DOCKER_ORGANIZATION_NAME/$DOCKER_REPOSITORY_NAME:${{ github.event.inputs.version }}-dev \
+          --file Dockerfile.release
+          --push .
```

```suggestion
      - name: Build & push prod docker images
        run: |
          docker buildx build \
          --target prod \
          --platform linux/amd64,linux/arm64 \
          --tag $DOCKER_ORGANIZATION_NAME/$DOCKER_REPOSITORY_NAME:${{ steps.get-commit-tag.outputs.VERSION_TAG }} \
          --tag $DOCKER_ORGANIZATION_NAME/$DOCKER_REPOSITORY_NAME:${{ steps.get-prod-tags.outputs.LATEST }} \
          --file Dockerfile.release
          --push .

      - name: Build & push dev docker images
        run: |
          docker buildx build \
          --target dev \
          --platform linux/amd64,linux/arm64 \
          --tag $DOCKER_ORGANIZATION_NAME/$DOCKER_REPOSITORY_NAME:${{ steps.get-commit-tag.outputs.VERSION_TAG }}-dev \
          --file Dockerfile.release
          --push .
```

I think this will enable us to be compliant with versions from before this update